tests/test_fixtures.py

```diff
@@ -310,7 +310,7 @@ def __init__(self, aktualizr_secondary_exe, id, port=9050, primary_port=9040):
     sqldb_path = "{db_path}"
 
     [pacman]
-    type = "fake"
+    type = "none"
```

Comment:
+1 for getting rid of "fake". Are there other uses of it? And is "none" still the right name for this? Should we call it "binary" or is that too misleading since it is so naïve?

Reply:
There was acutally another instance in `config/posix-secondary.toml`, I will fix it in another push to this branch.

We can discuss changing the name but we should be careful and be consistent. Plus we'd have several docs to change and I'm afraid it would be quite hard to remove all these uses of "fake" painlessly.

Reply:
> We can discuss changing the name but we should be careful and be consistent. Plus we'd have several docs to change and I'm afraid it would be quite hard to remove all these uses of "fake" painlessly.

Fair point. Might be worth consulting with tech writers or product team before we make bigger changes. No need to do all that work right now for this PR.